### ISSUE TITLE
Unhandled exception in concurrent task!

### ISSUE THREAD

**Issue by grubles** [ksedgwic/clboss#157]
Hitting this error with CLN at 57d21206dbeb05f085e288b9324c7bb8d76b7180

```
2023-03-18T13:07:00.073Z DEBUG   lightningd: Calling htlc_accepted hook of plugin clboss
2023-03-18T13:07:00.076Z **BROKEN** plugin-clboss: HtlcAcceptor: Unexpected payload of htlc_accepted: [REDACTED]
```

Also getting this printed to console repeatedly:

```
Unhandled exception in concurrent task!
Incorrect type.
...main loop continuing...
```

**Comment by ksedgwic** [comment]
This is likely addressed by https://github.com/ZmnSCPxj/clboss/pull/162

**Comment by ksedgwic** [comment]
Fixed by #162
